pulser/sequence.py

```diff
@@ -913,8 +913,11 @@ def draw(
         draw_phase_area: bool = False,
         draw_interp_pts: bool = True,
         draw_phase_shifts: bool = False,
-        fig_name: str = None,
-        kwargs_savefig: dict = {},
+        draw_register: bool = False,
+        fig_name_seq: str = None,
+        fig_name_reg: str = None,
+        kwargs_savefig_seq: dict = {},
+        kwargs_savefig_reg: dict = {},
```

Comment:
This seems a bit excessive... But ok, let's leave it this way.

Reply:
In a previous version I left just one `fig_name`, and I saved two images with an added prefix `pulses_` and `register_`. 

If you think it's better I could revert to that.

Reply:
Ah right, now that I think about it the reason I separated them is because this way you can choose whether you want to save just the pulses, just the register, or both

Reply:
Yeah, I would go for that, just detail it in the docstring. Also share the `kwargs`, I guess

Reply:
> Ah right, now that I think about it the reason I separated them is because this way you can choose whether you want to save just the pulses, just the register, or both

Sorry, I didn't see this before replying. Since they are saved separately, that doesn't sound so bad to me.